### ISSUE TITLE
Hi! Question on Tor usage

### ISSUE THREAD

**Issue by LisiLisenok** [tchoulihan/bitmerchant#8]
I looked through your code. It looks like to use tor with bitmerchant the tor to be initialized. So, call of bitcoin.useTor(); to be added, say, in LocalWallet#setupWalletKit method and that's all to use tor. Am I right? Or something else to be done to use it? Did you test tor with bitmerchant?
Thank's

**Comment by tchoulihan** [comment]
That's correct, I could add a command-line switch for this, If I haven't already.

**Comment by LisiLisenok** [comment]
Yes, please add, it would be nice. Thank you

**Comment by tchoulihan** [comment]
Okay, I'll make a note to myself to do this, It will probably be after the holidays, for me to implement and test this. It should be easy, but I'm away from my dev machine atm.

**Comment by valsha** [comment]
Hi, any news about this feature? Thank you.

**Comment by dessalines** [comment]
This would require an update to bitcoinj, which I haven't touched in years, and have since focused on other projects. If there's enough interest I'll start working on it again.
